kodi_addon_checker/check_addon_branches.py

```diff
@@ -19,17 +19,20 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def check_for_existing_addon(report: Report, addon_path: str, all_repo_addons: dict, pr: bool,
-                             kodi_version: KodiVersion):
+def check_for_existing_addon(report: Report, addon: Addon, all_repo_addons: dict, args):
     """Check if addon submitted already exists or not
         :addon_path: path of the addon
```

Comment:
Can you align the docs?

Reply:
I don't see them misaligned, the description is just small and the last arg description is a bit bigger. Is this what you mean?

Reply:
`addon` is missing. `addon_path` -> `report` and needs explanation. Not sure if `args` should get any docs.